pkg/cli/config/config.go

```diff
@@ -96,6 +98,52 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	return bpConfig, nil
 }
 
+func getProxyURL(bpConfig BackplaneConfiguration, proxyURLConfig any) string {
```

Comment:
you may not need to pass `bpConfig BackplaneConfiguration`  to return valid proxy.  what if implement the method as follows 
```
 func (config *BackplaneConfiguration) getValidProxy() {
    ... check if the proxy URL is a string or array here .....
 }
```

Reply:
Done